tool/gfm_stats.json

```diff
@@ -34,21 +34,21 @@
   "589": "strict",
   "590": "loose",
   "591": "loose",
-  "592": "loose",
```

Comment:
Trading failures is less than ideal...but our total numbers are better.

Reply:
If you look at the test cases for 592 and 595, you'll see that there's an incompatibility between the GFM autolink extensions and the original CM autolinks. 

Example 592 specifies that `< http://foo.bar >` should not get matched as a link, but the GFM autolink extension doesn't specify anything that would prevent that from matching.

Example 595 is literally just a link `http://example.com`, which obviously should be found by GFM autolinks.

Reply:
On that note, I'm authoring an issue right now on the [github/cmark](https://github.com/github/cmark) repo to address this. 

@srawlins You had mentioned offline that there were a few other projects using the cmark-gfm examples as test cases. Can you link me to those?

Reply:
Er, I thought pulldown-cmark (Rust) was... but they appear to be [using regular CommonMark](https://github.com/google/pulldown-cmark/blob/8b211fde18df2b974163716189cd7b197bdbc6b4/src/passes.rs#L100) tests. I'm not sure of others. In any case, I think it would be hard for them to argue that the examples should remain.

Reply:
@kevmoo I'd like to comment somewhere that 2 tests are failing because of a bug in github/cmark's tests, but not sure where to comment on that... maybe we can all just remember in our brains.

Reply:
[Here's the issue I opened with github/cmark](https://github.com/github/cmark/issues/81).

Reply:
Thanks that works.